### ISSUE TITLE
Use of proper database access library

### ISSUE THREAD

**Issue by krivachy** [danieljharvey/scala-egg-server#2]
Check them out: https://medium.com/@takezoe/database-access-libraries-in-scala-7aa7590aa3db

**Comment by krivachy** [comment]
Probably [Quill](http://getquill.io/) is a nice mix of lightweight + typesafe for you. Defacto these days is Slick, but it's a bit more complex to get running.

**Comment by danieljharvey** [comment]
Indeed. Found this which seems to have the basics reasonably well explained... https://blog.vogonjeltz.com/scala/programming/open-source/mysql/2016/08/19/actually-getting-data-with-quill.html
